code/modules/clothing/spacesuits/rig/suits/ert.dm

```diff
@@ -8,6 +8,7 @@
 	suit_type = "ERT commander"
 	icon_state = "ert_commander_rig"
 	emp_protection = 35
+	slowdown = 1
```

Comment:
that's not what I asked, I want you to move this line to `/obj/item/weapon/rig/ert` definition in rig.dm file, so rigs by default have slowdown of 1.

Reply:
I know what you asked. What Fowl said.

Reply:
You can't use scope of PR to justify ugliness of implementation